File: Enqueue_Front.c
void enqueueFront(struct Queue *queue, int value) {
    if (isFull(queue)) {
        printf("Queue is full. Cannot enqueue.\n");
    } else {
        if (isEmpty(queue)) {
            queue->front = 0;
            queue->rear = 0;
        } else {
            queue->front = (queue->front - 1 + MAX_SIZE) % MAX_SIZE;
        }
        queue->array[queue->front] = value;
    }
}

File: Enqueue_Front.py
def enqueueFront(self, value):
    if self.isFull():
        print("Queue is full. Cannot enqueue.")
    else:
        if self.isEmpty():
            self.front = 0
            self.rear = 0
        else:
            self.front = (self.front - 1 + MAX_SIZE) % MAX_SIZE
        self.array.insert(self.front, value)
